src/main/java/uk/gov/hmcts/cmc/claimstore/appinsights/AppInsightsTool.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.cmc.claimstore.appinsights;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+
+import static uk.gov.hmcts.cmc.claimstore.appinsights.AppInsights.REFERENCE_NUMBER;
+
+@Component @ConditionalOnProperty(prefix = "appinsightstool", name = "true")
+public class AppInsightsTool implements CommandLineRunner {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AppInsightsTool.class);
+
+    @Autowired
+    private AppInsights appInsights;
+
+    @Override
+    public void run(String... args) throws Exception {
+        LOGGER.info("Tracking events");
+        trackEvent(AppInsightsEvent.PAID_IN_FULL, 10); //1103);
```

Comment:
What is this value `10` here? I presume this is number of events to be fired. How did we calculated? How this number will be calculated in each env, specifically prod?

Reply:
This number is the first z amount of 'paid in full' claims, and these are being back filled. I've changed it locally as it needed to be as up-to-date as possible before I ran it. Will push the updated code now 👍 

Calculation:
```
Total number of paid in full claims (as of 29th Jan): x records
Custom paid in full events on App Insight: y events
x - y = z
z is the number in the code, aka the first z amount of 'paid in full' events to be backfilled

Expecting: x paid in full custom events on cmc-prod App Insight (success)
```